rs.ruffle.Ruffle.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: rs.ruffle.Ruffle
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ruffle
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: ruffle
+    buildsystem: simple
+    build-commands:
+      - install -D -t "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/" ./rs.ruffle.Ruffle.svg
+      - install -D -t "${FLATPAK_DEST}/share/applications/" ./rs.ruffle.Ruffle.desktop
+      - install -D -t "${FLATPAK_DEST}/share/metainfo/" ./rs.ruffle.Ruffle.metainfo.xml
+      - install -D -t "${FLATPAK_DEST}/bin/" ./ruffle
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/ruffle-rs/ruffle/efe754cdcc57f0e1f2820e66983f6a5a91e5d2c8/desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml
+        sha256: 07116dbdc54cf80ae4195337c8322022a6d3b3acdb37fef23bb1e3da9f1787c2
+        dest-filename: rs.ruffle.Ruffle.metainfo.xml
+      - type: file
+        url: https://raw.githubusercontent.com/ruffle-rs/ruffle/efe754cdcc57f0e1f2820e66983f6a5a91e5d2c8/desktop/packages/linux/rs.ruffle.Ruffle.desktop
+        sha256: a9929d86280116ddbe15c20a5a7ada95f6dca2226a54a16be298d34e1b264c10
+        dest-filename: rs.ruffle.Ruffle.desktop
+      - type: file
+        url: https://raw.githubusercontent.com/ruffle-rs/ruffle/efe754cdcc57f0e1f2820e66983f6a5a91e5d2c8/desktop/assets/icon.svg
+        sha256: 9ad3452f957260c4b3e23cc9e3b1a6da05cfa36cde2a584031c4a1f3aba15635
+        dest-filename: rs.ruffle.Ruffle.svg
+      # TODO Remove this patch when stable releases are present
+      - type: patch
+        path: patches/release.patch
+      - type: archive
+        url: https://github.com/ruffle-rs/ruffle/releases/download/nightly-2024-08-03/ruffle-nightly-2024_08_03-linux-x86_64.tar.gz
```

Comment:
why no building from source instead?

Reply:
That's something I'm trying to wrap my head around. I wasn't sure how to do it and just left downloading binaries as in the original PR.

I saw https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo, but it seems that it requires manually generating the JSON each release in flathub's repo or publishing it upstream, right?

Reply:
Okay, I think I got it! Ruffle is being built from sources. Let me know if everything's alright now